README.md

```diff
@@ -531,96 +567,108 @@ public class Rectangle implements Shape {
+`nano`  | A easy-to-use command line text editor
+`cat`  | conCATenates/combines - shows you the file
+`cp`  | CoPy file
+`mv`  | MoVe file or directory. In Bash moving and renaming are the same operation
+`rm`  | ReMove file - be VERY careful with this one, as files do not end up in the trash and are gone forever!
+
+Concept | description
+----------------------
+`/`  | the 'root' directory. The highest directory you can navigate to; all other directories on your computer are somewhere beneath this one
+`~`  | the home directory
+`.`  | the current directory
+`..`  | the directory above the current one
+`-`  | the previous directory you navigated to
+
+### Git commands
+
+All of these commands must begin with `git` followed by a space.
+
+Command | description
+---------------------
+`status`  | checking the state of a repository
+`init`  | creates an repository with no commits from the current directory
+`add`  | adds files where the names succeed the key word
+`commit -m "<commit message here>"` | commits the files to the repository. The `-m` flag is needed so you don't open up your shell editor
 
 ## Git Repositories
 
-We create repositories in Git to initialise a version control system and allow many people to work on the same code.
-
-To create a repository, 'cd' to the directory you want to create the repository in.\
-Use 'ls' to see everything that the directory contains.\
-Create the repository using 'git init'.\
-Use 'git status' to check the status of the repository.\
-Use 'git add' followed by the full file names, or '.' for everything, to prepare files for adding to the repository.\
-Use 'git status' to check the status of the repository.\
-Use 'git commit' to commit (add) the files to the repository. Use '-m' followed by text in "" to add comments to the commit.\
-Comments are very good for explaining why you are doing things.\
-Use 'git status' to check the status of the repository.\
-If there are files you want to be ignored or you want to create a space for future files to ignore, type 'nano' followed by a file name to open the nano text editor and load a file. If no file is found nano will create one.\
-Use 'git status' to check the status of the repository.\
-Use 'git add' to prepare the to-be-ignored files for adding to the repository.\
-Use 'git status' to check the status of the repository.\
-Use 'git commit' to commit (add) the files to-be-ignored to the repository. Use '-m' followed by text in "" to add comments to the commit.\
-Use 'git status' to check the status of the repository.\
-When you are done, you should get the message 'working tree clean'.
+We create repositories in Git to initialise a version control system and track the changes that are being made to the files within the directory the repository is made from.
+
+1. To create a repository, `cd` to the directory you want to create the repository in.
+1. Use `ls` to see everything that the directory contains.
+1. 1. Create the repository using `git init`.
```

Comment:
Did you mean to put two '1.' here? I tried this in markdown preview and it doesn't seem to form a sub-list.

Reply:
Nope. This was unintentional. I'm deleting the redundant 1.

Although it doesn't form a sublist in VS Code, it [DOES do this in GitHub](https://github.com/JStonham/notes/blob/afaf8dc650798d8641161b08d2e87e053ae8fe49/README.md#git-repositories). Remember how I mentioned that GitHub flavoured markdown has some extra features? This might be one of them. Granted, it looks silly.